src/components/SomePage.spec.tsx

```diff
@@ -17,23 +17,22 @@ describe("<SomePage/>", () => {
     expect(button).not.toBeDisabled();
   });
 
-  it("test 2", async () => {
-    const user = userEvent.setUp();
+  it("test 4", async () => {
     render(
       <SomePage/>
     );
     const button = screen.getByText("Click me");
-    user.click(button);
+    await userEvent.click(button);
```

Comment:
@coderabbitai ensure calls to userEvent.click are always wrapped in a call to waitFor